src/components/auth/checkEmail/CheckEmail.tsx

```diff
@@ -0,0 +1,28 @@
+import { JSX } from 'react'
+
+import s from './CheckEmail.module.scss'
+
+import { Email } from '@/assets'
+import { TypographyVariant } from '@/common'
+import { Button, Card, Typography } from '@/components'
+
+type CheckEmailProps = {
+  email: string
+}
+
+export const CheckEmail = ({ email = 'example@mail.com' }: CheckEmailProps): JSX.Element => {
+  return (
+    <Card className={s.formWrapper}>
+      <Typography className={s.formHeader} variant={TypographyVariant.Large} as="h2">
+        Check Email
+      </Typography>
+      <Email className={s.emailImage} />
+      <Typography className={s.instructionText} as="p" variant={TypographyVariant.Body2}>
```

Comment:
p is default value in Typography, remove it

Reply:
Done!